test/validators/movie.test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
```

Comment:
in line with the above comment, this file should become test/validators/movies/create.test.js